src/java.base/windows/native/libjava/jlong_md.h

```diff
@@ -63,15 +63,8 @@
 /* For static variables initialized to zero */
 #define jlong_zero_init ((jlong) 0)
 
-#ifdef _WIN64
```

Comment:
After this change I think the differences between the unix and windows variants of this file are trivial
and could be resolved in favor of moving everything directly into jlong.h.  Though note there are some
places in java.desktop that currently directly include jlong_md.h.  This can be done as a separate cleanup.

Reply:
Right. I updated JDK-8343291 to cover this case as well.